README.md

```diff
@@ -1,4 +1,4 @@
-# Apps Catalog
+\*\*\*\*# Apps Catalog
```

Comment:
Is this intentional?

Reply:
Nope :D
Keyboard Shortcuts inside markdown are different